smartprom.py

```diff
@@ -74,7 +74,7 @@ def get_smart_status(results: dict) -> int:
 
 def smart_sat(dev: str) -> dict:
     """
-    Runs the smartctl command on a "sat" device
+    Runs the smartctl command on a internal or external "sat" device
     and processes its attributes
     """
     results, exit_code = run_smartctl_cmd(['smartctl', '-A', '-H', '-d', 'sat', '--json=c', dev])
```

Comment:
Are you sure it works with the 'sat' type in the smarctl command instead of 'usbprolific'?

Reply:
It works for both 'sat' and 'usbprolific'.
Below is the command execution result in my environment, the output result of 'sat' and 'usbprolific' is the same.

> $ smartctl --scan
> /dev/sdd -d usbprolific # /dev/sdd [USB Prolific], ATA device
> $ sudo smartctl -A -d sat /dev/sdd
> $ sudo smartctl -A -d usbprolific /dev/sdd